types/options.d.ts

```diff
@@ -0,0 +1,15 @@
+export interface Options {
```

Comment:
I'm approving but we'll need to move this and [this](https://github.com/ripe-tech/ripe-twitch-api-js/blob/master/types/order.d.ts) somewhere around here: https://github.com/hivesolutions/yonius/tree/master/types/data

@BeeMargarida @joao-conde one of you please pick this up.